2-multitenant/README.md

```diff
@@ -1,2 +1,54 @@
 # Multitenant Infrastructure phase
-This phase deploys multitenant infrastruture using the multitenant infrastructure pipeline.
+
+## Purpose
+
+This phase deploys the per-environment multitenant resources deployed via the multitenant infrastructure pipeline.
+
+The following resources are created:
+- GCP Projects
+- GKE cluster(s)
+- Cloud SQL PostgreSQL
```

Comment:
Could you explain why a DB is created here? One could argue a DB should belong to one of the app phase?

Reply:
This PR is primarily `as-built` documentation, but I'll inquire if it can be moved to the app phase as I'd also prefer the architecture.

Reply:
These will be moved to phase 4 in a different PR.

Reply:
https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/pull/96